site/src/components/card.md

```diff
@@ -0,0 +1,203 @@
+<article class="card width-300">
+	<div class="card-content">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<p class="card-content-description">Additonal details might be listed here.</p>
+	</div>
+</article>
+```
+
+## Card footer
+
+The `.card-footer` class is an optional. It is often used to display actions related to the card's subject (like "save" or "follow"). Or for display information that doesn't fit well elsewhere, such as progress. It has a child component called `.card-footer-item`. Place `.card-footer-left` or `.card-footer-right` on `.card-footer` (or use flex atomics) to align its inner items as you see fit. Note that the `.buttons` class is well supported in this context and can be added if space between buttons is desired. It's recommended that you use small buttons (`.button-sm`) if placing buttons in this area.
+
+```html
+<article class="card width-350">
+	<div class="card-content">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<p class="card-content-description">Additonal details might be listed here.</p>
+	</div>
+	<div class="card-footer">
+		<div class="card-footer-item">
+			<progress style="width: 60px;" max="100" value="70"></progress>
+		</div>
+		<div class="card-footer-item">
+			<div class="buttons">
+				<button class="button button-clear button-primary button-sm">Share</button>
+				<button class="button button-primary button-filled button-sm">Save</button>
+			</div>
+		</div>
+	</div>
+</article>
+```
+
+## The card header
+
+The optional `.card-header` class can be used as a container to display anything you'd like pinned at the top of the card. It has only one possible subcomponent, `.card-header-image`. The example below uses several atomics to create a flavorful pattern and tag. Note that this is not a recommended pattern, just an example of the possibilties.
+
+```html
+<article class="card width-350">
+	<div
+		class="card-header position-relative background-size-100 background-image-pattern background-color-body-accent"
+		style="--background-image-pattern: url('https://docs.microsoft.com/en-us/media/background-patterns/plus.svg')"
+	>
+		<div class="card-header-image">
+			<img
+				aria-hidden="true"
+				class="card-template-icon"
+				src="https://docs.microsoft.com/en-us/learn/achievements/review-microsoft-azure-pricing-slas-lifecycles.svg"
+			/>
+		</div>
+		<div class="position-absolute top-0 right-0 margin-top-xs margin-right-xxs">
+			<span class="border-radius  background-color-body-accent padding-xxs border">NEW</span>
+		</div>
+	</div>
+	<div class="card-content">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+	</div>
+</article>
+```
+
+## Wider cards
+
+On certain views, it may be desireable for a particular card to take up a wider space. There are two options for such views.
```

Comment:
Isn't it the default case?

Reply:
Please read on, this is discussed in next paragraph